src/app/shared/components/cluster-summary/template.html

```diff
@@ -116,6 +116,9 @@
       <km-property-boolean label="OPA Integration"
                            [value]="cluster.spec.opaIntegration?.enabled">
       </km-property-boolean>
+      <km-property-boolean label="Konnectivity"
+                           [value]="cluster.spec.clusterNetwork.KonnectivityEnabled">
```

Comment:
Please add a `?` next to `clusterNetwork` -> `cluster.spec.clusterNetwork?.KonnectivityEnabled` (also wondering about upper case letter, but that's probably answered in the other comment 😉 )
Otherwise UI will throw errors.